packages/hash/external-services/kratos/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM oryd/kratos:v0.10.1
+
+USER root
+
+RUN sed -i 's|v3\.\d*|edge|' /etc/apk/repositories
+RUN apk update
+RUN apk add yq~=4.30
+
+# This can be either `dev` or `prod`
+ARG ENV=dev
+ENV ENV=$ENV
+ARG API_SECRET=secret
+ENV API_SECRET=$API_SECRET
+
+RUN mkdir -p /etc/config/kratos && \
+    mkdir -p /home/ory/.postgresql
+
+COPY ./hooks /etc/config/kratos/hooks
+COPY ./identity.schema.json /etc/config/kratos/
+COPY kratos.$ENV.yml /etc/config/kratos/kratos.$ENV.yml
+# Change the API key secret for the webhook 
+RUN yq -i '.selfservice.flows.registration.after.password.hooks[0].config.auth.config.value |= strenv(API_SECRET)' /etc/config/kratos/kratos.$ENV.yml
+
+RUN ln /etc/config/kratos/kratos.$ENV.yml /etc/config/kratos/kratos.yml
```

Comment:
Just for my understanding, the process of updating the kratos configuration is as simple as rebuilding this container and replacing the running one in production?

Reply:
Yes exactly. This is how it's done currently in production. It may not be best practise to put the secret into the container, but there still isn't a way to pass it as an ENV variable from what i can tell